platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -91,12 +81,18 @@ export interface PolyIn {
     add(...quads: RDF.Quad[]): Promise<void>;
 
     /**
+     * Deletes the indicated triples
+     *
      * @param quads the triples that should be removed from the Pod
      */
     delete(...quads: RDF.Quad[]): Promise<void>;
 
     /**
+     * Checks whether the set of triples (called quads because they include the graph or namespace)
+     * are included in the pod. Returns true if they do.
+     *
      * @param quads the triples that should be removed from the Pod
+     * @returns a Promise that will be resolved to a boolean.
```

Comment:
The docs always ignore promises, I think we should keep that up. So here in the docs we can pretend the return value is that boolean.